TESTS/netsocket/tcp/tcpsocket_bind_address.cpp

```diff
@@ -26,6 +26,7 @@ using namespace utest::v1;
 
 void TCPSOCKET_BIND_ADDRESS()
 {
+    SKIP_IF_TCP_UNSUPPORTED
```

Comment:
Is there a reason this isn't 
```suggestion
    SKIP_IF_TCP_UNSUPPORTED();
```

instead?

Reply:
I guess you're right - after all this macro calls another macro, so perhaps it's complex enough to earn an empty parameter list :)

Reply:
Making it look like a function call will improve the readability.